src/org/gk/gkCurator/ConstantsManager.java

```diff
@@ -0,0 +1,23 @@
+package org.gk.gkCurator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConstantsManager {
```

Comment:
Is there an advantage to use this class? I guess not. If it is really needed, it should be configured by an outside file. However, I don't think there is a need for this. Just using these Strings directly when they are needed will be simple and easy to maintain.

Reply:
Sounds good! I will move the strings to where they are used directly and delete the class.